apps/dashboard/app/models/motd_formatter_plaintext_erb.rb

```diff
@@ -0,0 +1,20 @@
+class MotdFormatterPlaintextErb
+
+  attr_reader :content, :title
+
+  def initialize(motd_file)
+    motd_file = MotdFile.new unless motd_file
+    @title = motd_file.title
+    file_content =
+    	begin
+    		ERB.new(motd_file.content).result
+    	rescue
+    		motd_file.content
+    	end
+    @content = file_content

```

Comment:
The rendering should be moved to another method:

```
def render(content)
  ERB.new(motd_file.content).result
rescue
  motd_file.content
end
```

@johrstrom thoughts on the rendering context for this? I know that in other places we do ERB rendering a little different than `ERB.new(motd_file.content).result`.

Reply:
Also be careful about not introducing tabs. Only spaces should be used.

Reply:
Yea I think a helpful context is more about libraries/functions than variables.

Maybe with stuff like this
```ruby
def groups
  groups = OodSupport::Process.groups.map(&:name)
end
```

I think a lot of use cases are going to revolve around disk space, queue sizes and billing. Maybe we should add a class in `lib/` that starts simple and we can just build helpers into it as needed. Also - folks could initialize it to put in there way from their MOTD file.

Reply:
If we add a new class and move a general purpose rendering function out of this, I think I'd like to do so by identifying 3 instances of this rendering we want to move to this new class. So this PR is one... what are the others?

Reply:
I found a number of instances for this rendering, 
```
[kalattar@owens-login01 dashboard-motd-link]$  ack 'ERB.new'
app/models/announcement.rb
43:                  YAML.safe_load(ERB.new(@path.expand_path.read, nil, "-").result)

app/models/batch_connect/app.rb
286:        erb = ERB.new(contents, nil, "-")

app/models/batch_connect/session.rb
497:          rendered = ERB.new(template, nil, "-").result(binding)

app/views/batch_connect/sessions/connections/_custom.html.erb
1:<%= OodAppkit.markdown.render(ERB.new(view, nil, "-").result(connect.instance_eval { binding })).html_safe %>

app/views/batch_connect/sessions/connections/_info.html.erb
1:<%= OodAppkit.markdown.render(ERB.new(view, nil, "-").result(session.instance_eval { binding })).html_safe %>

```

Reply:
Since we have a lot of MOTD classes, how about moving them all to a module and adding some `module_function`s?  I guess I would advocate less for context state in the rendering and more about easy access to helper functions. Helper functions like `groups` above provide value, not necessarily a classes' instance variables.

Though we should probably just defer the refactor/additions now that we've identified that we want them but don't know how to implement it.

Reply:
That sounds good, I can start implementing that, and do a PR on it on another branch.